src/redux/rockets/reducer.js

```diff
@@ -0,0 +1,38 @@
+import axios from 'axios';
+import * as actions from './actions';
+
+const stateInit = [];
+const url = axios.create({ baseURL: 'https://api.spacexdata.com/v3/rockets' });
+
+const rocketsReducer = (state = stateInit, action) => {
+  let d;
+  let x;
+  switch (action.type) {
+    case actions.ADD_RESERVATION: {
+      const nextState = state.map((rocket) => (
+        (rocket.id !== action.payload) ? rocket : { ...rocket, reserv: true }));
+      d = nextState;
```

Comment:
[Optional] In my opinion, it's better to use the code logic inside the thunk, and simply return the `action.payload` from our reducer. This way our code will look much cleaner.